elixir/test/extism_test.exs

```diff
@@ -0,0 +1,54 @@
+defmodule ExtismTest do
+  use ExUnit.Case
+  doctest Extism
+
+  # test "context create & reset" do
+  #   ctx = Extism.Context.new()
+  #   Extism.Context.reset(ctx)
+  # end
+
+  # defp new_plugin() do
+  #   ctx = Extism.Context.new()
+  #   path = Path.join([__DIR__, "../../wasm/code.wasm"])
+  #   manifest = %{wasm: [%{path: path}]}
+  #   {:ok, plugin} = Extism.Context.new_plugin(ctx, manifest, false)
+  #   plugin
+  # end
+
+  test "counts vowels" do
+    Extism.set_log_file("/tmp/extism.log", "DEBUG")
+    ctx = Extism.Context.new()
+    path = Path.join([__DIR__, "../../wasm/code.wasm"])
+    manifest = %{wasm: [%{path: path}]}
+    {:ok, plugin} = Extism.Context.new_plugin(ctx, manifest, false)
+    {:ok, plugin} = Extism.Context.new_plugin(ctx, manifest, false)
+    {:ok, plugin} = Extism.Context.new_plugin(ctx, manifest, false)
+    {:ok, output} = Extism.Plugin.call(plugin, "count_vowels", "this is a test")
```

Comment:
Tried creating a few plugins and saw the plugin_id increment. But when trying to call i get the plugin with id doesn't exist panic.